.travis.yml

```diff
@@ -1,37 +1,60 @@
 language: cpp
+#services:
+#  - docker
+
+arch:
+  - amd64
+  - arm64
 
 os:
   - linux
   - osx
+  - windows
 
 compiler:
   - gcc
 
+matrix:
+  fast_finish: true
+  include:
+    - os: linux
+      arch: amd64
+    - os: linux
+      arch: arm64
+    - os: windows
+    - os: osx
+
 addons:
   apt:
     packages:
     - libgtk-3-dev
     - nasm
+    - build-essential
+    - cmake
+    - libjpeg-dev
 
 before_script:
+#  - docker build --tag openpnp-capture-rpi3-arm64:latest . -f docker/Dockerfile.rpi3_arm64 
   - mkdir build
   - cd build
   - cmake ..
 
 script: make
+# docker run -v $PWD:/openpnp-capture openpnp-capture-rpi3-arm64 make
```

Comment:
Note that make won't work for Windows. See the appveyor.xml for info on how we build there. We may need to make changes for Travis.

Reply:
Yeah, I know, my intention was to go for this: https://docs.travis-ci.com/user/reference/windows/#how-do-i-use-msys2 ... I've seen the appveyor.xml, first I wanted to get some red though, I'll trigger it in the next few minutes...